docker-compose.yml

```diff
@@ -31,7 +34,7 @@ services:
     volumes:
       - jquerydbdata:/var/lib/mysql
     ports:
-      - "${JQUERY_WP_MYSQL_PORT:-9414}:3306"
+      - "3306:3306"
```

Comment:
3306 is the default MySQL port. If you install mysql (e.g. apt-get or brew), or have any other project with a similar docker container, this immediately clashes.

Unlike http/https, the mysql port doesn't actually need to be exposed to the outside. We could remove it completely, or (my suggestion) to leave it with a custom value as the default.

It's sole purpose is to allow you to explore the database through some GUI app or CLI tool that you fancy from your own machine to be able to connect to it.

Reply:
Actually, not exposing it at all makes the most sense to me.